netcontrold/lib/dataif.py

```diff
@@ -568,8 +569,7 @@ def pmd_load(pmd):
     # Given we have samples of rx packtes, processing and idle cpu
     # cycles of a pmd, calculate load on this pmd.
     # sort counters so that, incremental differences calculated.
-    sort_rx_cyc = pmd.rx_cyc[:]
-    sort_rx_cyc.sort()
+    sort_rx_cyc = sorted(pmd.rx_cyc[:])
```

Comment:
do we use this sort_rx_cyc elsewhere ?

Reply:
This change is also made by autopep8

Reply:
i dont think so. Could you double check on it ?.

Reply:
ya ok

Reply:
got it.